scene/gui/line_edit.cpp

```diff
@@ -698,6 +698,7 @@ void LineEdit::_notification(int p_what) {
 		case NOTIFICATION_DRAW: {
 
 			if ((!has_focus() && !menu->has_focus()) || !window_has_focus) {
+				selection = Selection();
```

Comment:
Need to call `deselect()` instead.

Reply:
Done... my bad for not seeing that method!